app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -503,6 +504,13 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             )
         )
 
+        if (components.core.store.state.findActiveMediaTab() != null) {
+            lifecycleScope.launch(IO) {
```

Comment:
Not sure this is really needed. Have you observed an adverse effect of running this without the coroutine?
It's just that we already filter tabs in many (like it happens with `findActiveMediaTab()` on the Main thread without a known impact and there is also an inherent cost for the context switching here so I'm not sure of the benefits.

Reply:
I didn't observe any improvement using the coroutine. You're right regarding the performance penalty with context-switching. I'll remove the coroutine.